plugins/lookup/tss.py

```diff
@@ -82,6 +82,18 @@
                 | items2dict(key_name='slug',
                              value_name='itemValue'))['password']
             }}
+            
+- hosts: localhost
+  vars:
+      secret: "{{ lookup('community.general.tss', 102, base_url='https://secretserver.domain.com/SecretServer/', username='user.name', password='password') }}"
```

Comment:
A potential multi-line version could be:
```suggestion
      secret: >-
          {{ lookup('community.general.tss', 102,
                  base_url='https://secretserver.domain.com/SecretServer/',
                  username='user.name', password='password') }}"
```